### PR TITLE
MINOR: Reduce commit time on test

### PR DESCRIPTION
**Pull request by bbejeck**
Reduce on commit time in the test.

The test fails as it can't read all sent messages within 30 seconds, I've reduced commit interval to help mitigate timing.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/StreamToTableJoinScalaIntegrationTestImplicitSerdes.scala [hunk 1]
```diff
@@ -18,22 +18,19 @@ package org.apache.kafka.streams.scala
...
-
-import org.apache.kafka.streams.integration.utils.{EmbeddedKafkaCluster, IntegrationTestUtils}
 import org.apache.kafka.clients.consumer.ConsumerConfig
 import org.apache.kafka.clients.producer.ProducerConfig
-
 import org.apache.kafka.common.serialization._
 import org.apache.kafka.common.utils.MockTime
-import org.apache.kafka.test.TestUtils
 import org.apache.kafka.streams._
+import org.apache.kafka.streams.integration.utils.{EmbeddedKafkaCluster, IntegrationTestUtils}
```
Comment (bbejeck) [inline streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/StreamToTableJoinScalaIntegrationTestImplicitSerdes.scala:26]: Import changes from auto Intellij setting, I can revert if wanted.

**Comment by bbejeck** [comment]
@hachikuji 
\cc @guozhangwang @mjsax @vvcephei

**Review by mjsax — APPROVED** [review]
LGTM

**Comment by guozhangwang** [comment]
LGTM!

**Pull request merged**